Draft d'une question sur les avantages de vacciner un enfant

DuniaAch opened the pull request:
https://www.education.gouv.fr/covid19-la-vaccination-des-12-17-ans-323573

DuniaAch committed:
Draft d'une question sur les avantages de vacciner un enfant

https://www.education.gouv.fr/covid19-la-vaccination-des-12-17-ans-323573

DuniaAch committed:
Update 8-conseils-pour-les-enfants.md

ronnix committed:
Relecture vacciner mon enfant

ronnix committed:
Typographie

ronnix committed:
Relecture vacciner mon enfant

ronnix commented:
:+1:

davidbgk commented:
> L’infection par le SARS-CoV-2 épargne relativement les adolescents, rarement symptomatiques. Il existe un gradient de susceptibilité et d’infectivité croissant avec l’âge. Les adolescents développent peu de formes graves de la Covid-19. Par ailleurs, il semble que dans 45 à 75% des cas de formes sévères chez les enfants/adolescents, une comorbidité soit associée.
> 
> Une forme spécifique, les « syndromes inflammatoires multi-systémiques pédiatriques » ou MIS-C ou PIMS, grave mais rarement mortelle, a été décrite chez l’enfant (parmi les 558 PIMS signalés, 486 avait un lien confirmé, probable ou possible avec la COVID-19). Elle touche toutefois des enfants jeunes (âge médian = 8 ans) et concerne donc dans une moindre mesure les adolescents. 
>
> En France, 4295 hospitalisations et 737 admissions en soins critiques ont été répertoriées depuis mars 2020 pour la population de moins de 18 ans (qui représentent 1,1% du total des hospitalisations et 0,9% du total des admissions en soins critiques). Chez les adolescents âgés de 11 à 17 ans, la part des hospitalisations liées à la Covid19 est de 0,44% et celle des admissions en soins critiques de 0,36%.
>
> — https://www.has-sante.fr/upload/docs/application/pdf/2021-06/strategie_de_vaccination_contre_la_covid-19_-_place_du_vaccin_a_arnm_comirnaty_chez_les_12-15_ans_-_recommandation.pdf

On gagnerait peut-être à documenter/argumenter sur certaines parties 🤔

DuniaAch committed:
Ajoute les chiffres liés à l'hospitalisation des ados

Sur proposition de @davidbgk 

Je vais faire une question sur les manifestations plus graves

davidbgk commented on contenus/thematiques/8-conseils-pour-les-enfants.md:
```diff
@@ -351,6 +351,16 @@ Le forfait « *100% Psy Enfant Ado* » donne accès à 10 séances de **souti
+
+    Il est vrai que les jeunes présentent le plus souvent des formes bégnines de Covid (la part des hospitalisations des 11 - 17 ans est de 0,44 % et celle des admissions en soins critiques de 0,36 %), mais ils peuvent parfois développer des cas plus sévères ou souffrir de symptômes persistants (Covid long). Le vaccin permet alors de limiter efficacement ces risques.
+    
+    En se vaccinant, les jeunes participent surtout à l’effort collectif de **santé publique :**
```
```suggestion
    En se vaccinant, les jeunes participent surtout à l’effort collectif de **santé publique** :
```

davidbgk committed:
Relecture vaccination enfants
